src/alpha-lab/blink_detection.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Detecting eye blinks using Pupil Lab’s blink detection pipeline"
+description: Perform posthoc and real-time blink detection
+permalink: /alpha-lab/blink-detection/
+tags: [Neon, Realtime Python API]
+---
+
+# Detecting eye blinks using Pupil Lab's blink detection pipeline
+
+<font size = 2>Products used: [Neon](https://pupil-labs.com/products/neon/), [Realtime Python API](https://pupil-labs-realtime-api.readthedocs.io/en/stable/)</font>
+
+<div class="pb-4" style="display:flex;justify-content:left;">
+  <v-img
+    class="rounded"
+    :src="require('../media/alpha-lab/eye_blinks_anim.gif')"
+    max-width=800px
+  >
+  </v-img>
+</div>
+
+### Eye blinks: artifacts and information
+
+The accurate detection of eye blinks serves a vital role in numerous eye tracking applications. Its importance is twofold: firstly, it helps identify time periods affected by blink-related artifacts, and secondly, isolate key physiological, cognitive, and clinical parameters, such as blink rate and blink duration. Blink detection can even be utilized in order to control external devices, for example by detecting a sequence of blinks made in quick succession. In order to show you how to achieve all of this, we have put together this guide.
+
+### Running blink detection locally
+
+Eye blinks are automatically detected once your recording is uploaded to Pupil Cloud. This guide shows how to apply Pupil Labs' blink detection algorithm to Neon recordings <i>offline</i> (i.e., without using Pupil Cloud) or <i>in real-time</i> (using [Pupil Lab's Realtime Python API](https://github.com/pupil-labs/realtime-network-api)). This allows you to build novel applications or simply satisfy your curiosity and develop a deeper understanding of the underlying algorithms.
```

Comment:
Can we just call them "blinks" instead of "eye blinks"? Is there a reasoning for this?